### PR TITLE
Build on py2

### PR DESCRIPTION
**Pull request by martindurant**

### CODE REVIEW

**Comment by martindurant** [comment]
This is only building py3.4/5/6, even after removing the skip comment for py2. How do I enable py27 builds?

**Comment by mrocklin** [comment]
I note that some of the other yaml files only list Python 3 bits.  Perhaps there is some way to automatically change all of those.  Would `conda-smithy rerender` help?

cc @jakirkham , who might be able to help us here

**Comment by martindurant** [comment]
Thanks, @mrocklin , just came to realise that also.

File: .travis.yml [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
 
 os: osx
-osx_image: beta-xcode6.1
+osx_image: xcode6.4
 
 env:
   matrix:
     
-    - CONDA_PY=34
+    - CONDA_PY=27
```
Comment (mrocklin) [inline .travis.yml]: That's encouraging

**Comment by mrocklin** [comment]
cc @jbednar this may affect you

**Comment by jakirkham** [comment]
Yeah, as you may already realize, if one changes `skip`, one must also re-render for the change to propagate to the CI files. This also effects other subtle changes like metadata from the recipe that gets added to the feedstock's Readme.

**Pull request merged**